.eslintignore

```diff
@@ -4,3 +4,6 @@ node_modules
 **/build
 jupyterlab/staging/yarn.js
 jupyterlab/static
+
+dev_mode/index.js
+jupyterlab/staging/index.js
```

Comment:
I had to disable eslint on these 2 files because simply adding them to .prettierignore didn't ignore them from eslint.